deploy/templates/nstemplatetiers/appstudio/ns_tenant.yaml

```diff
@@ -156,7 +154,6 @@ objects:
     limits:
     - type: "Container"
       default:
-        cpu: 2000m
         memory: 2Gi
       defaultRequest:
         cpu: 10m
```

Comment:
Should we put 200m as default cpu request, as it was tested by performance team?

Reply:
Yes you're right.